qualtran/bloqs/hamiltonian_simulation/hamiltonian_simulation_by_gqsp_test.py

```diff
@@ -83,10 +82,9 @@ def verify_hamiltonian_simulation_by_gqsp(
     assert_matrices_almost_equal(expected_top_left, actual_top_left, atol=1e-4)
 
 
-@pytest.mark.slow
 @pytest.mark.parametrize("select_bitsize", [1])
 @pytest.mark.parametrize("target_bitsize", [1, 2])
-@pytest.mark.parametrize("t", [2, 3, 5])
+@pytest.mark.parametrize("t", [2, 3, pytest.param(5, marks=pytest.mark.slow)])
```

Comment:
oh nice! didn't know this feature

Reply:
me neither

that's also why I left in the skip from the above test that tanuj noted -- it's a nice example of how to skip particular configurations :)